docs/community/com-query/grafana-connector.md

```diff
@@ -8,23 +8,30 @@ nav_order: 2
 
 # Grafana Connector
 
-Molecula provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
+Featurebase provides a Grafana plugin, which can be used to interact with FeatureBase (4.0 or higher <!-- TODO is this correct? -->) to run queries and view visualizations of results. Note that this is unrelated to using Grafana for operational monitoring of the FeatureBase cluster. For details on that, see the [monitoring reference page](/docs/community/com-monitoring/old-monitoring)
 
 ## Setup
 
 ### Install Grafana
-For the best experience, [install](https://grafana.com/grafana/download) Grafana version 7 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
+For the best experience, [install](https://grafana.com/grafana/download) Grafana version 9 using the binary, not brew. The plugin requires version 7 or greater, but version 7 works best. For convenience, here are the commands for mac. For other platforms, check the link.
 
+For the best experience, [install](https://grafana.com/grafana/download) Grafana using the binary, not brew. The plugin requires version 7 or greater and works best on version 9.
+
+### Mac OS 
```

Comment:
Where should the user run this script?
Does the user need specific privileges on the system?
What results from running this script?

Reply:
@rachithrr you can skip this